Vagrantfile

```diff
@@ -0,0 +1,47 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64"
+
+  config.vm.provision "shell", privileged: false, inline: <<-SHELL
+
+  sudo apt-get update
```

Comment:
I would indent this section up to `SHELL` as it's just the value for the `inline` argument for the `config.vm.provision` function on line 4